### PR TITLE
RGB support

### LINKED ISSUE #56: RGB support
We should make sure we work with hyper-spectral / RGB images e.g. Ti-7

### PR DESCRIPTION
**Pull request by dzenanz**
Closes #56.

### CODE REVIEW

File: include/itkTileMergeImageFilter.hxx [hunk 1]
```diff
@@ -418,15 +421,16 @@ TileMergeImageFilter<TImageType, TInterpolator>
     this->UpdateProgress(0.0);
     for (unsigned i = 0; i < m_Regions.size(); i++)
       {
-      PixelType val = 0;
-      PixelType bits = sizeof(PixelType) * 8;
+      PixelType val = NumericTraits<PixelType>::Zero;
```
Comment (thewtex) [inline include/itkTileMergeImageFilter.hxx]: `ZeroValue()` is a best practice over `Zero` for multi-component support.

File: include/itkTileMergeImageFilter.hxx [hunk 2]
```diff
@@ -541,6 +545,8 @@ TileMergeImageFilter<TImageType, TInterpolator>
       tileRegions[t] = &m_InputMappings[tileIndices[t]];
       }
 
+    TPixelAccumulateType zeroSum = NumericTraits<TPixelAccumulateType>::Zero;
```
Comment (thewtex) [inline include/itkTileMergeImageFilter.hxx]: `ZeroValue()`

File: test/itkMontageTestHelper.hxx [hunk 3]
```diff
@@ -61,21 +61,23 @@ void WriteTransform(const TransformType* transform, std::string filename)
...
+  using ScalarPixelType = typename itk::NumericTraits<PixelType>::ValueType;
   constexpr unsigned Dimension = 2;
   using PointType = itk::Point<double, Dimension>;
   using VectorType = itk::Vector<double, Dimension>;
   using TransformType = itk::TranslationTransform<double, Dimension>;
-  using ImageType = itk::Image< PixelType, Dimension>;
-  using ImageTypePointer = typename ImageType::Pointer;
-  using PCMType = itk::PhaseCorrelationImageRegistrationMethod<ImageType, ImageType>;
+  using ScalarImageType = itk::Image< ScalarPixelType, Dimension>;
+  using OriginalImageType = itk::Image< PixelType, Dimension>; // possibly RGB instead of scalar
```
Comment (thewtex) [inline test/itkMontageTestHelper.hxx]: These are nice names :+1:

**Comment by dzenanz** [comment]
The CI fails because a newer version of ITK is required. Will this [PR](https://github.com/InsightSoftwareConsortium/ITKContinuousIntegration/pull/18) fix it?

**Comment by thewtex** [comment]
Sorry, I missed that connection...

Thanks for `ITKContinuousIntegration` PR. Restarting the build...

**Pull request merged**